src/auto_code_surveys.py

```diff
@@ -36,9 +37,27 @@ def auto_code_surveys(cls, user, data, icr_output_dir, coda_output_dir):
                 TracedDataCodaV2IO.export_traced_data_iterable_to_coda_2(
                     data, plan.raw_field, plan.time_field, plan.id_field, {plan.coded_field: plan.code_scheme}, f
                 )
+
+        # Output messages for thematic analysis
+        IOUtils.ensure_dirs_exist(icr_output_dir)
+        for plan in PipelineConfiguration.SURVEY_CODING_PLANS:
```

Comment:
Why does this need to be incremental?

Reply:
The file output, you mean? Because we needed to start thematic analysis before the data collection concluded. Does that answer your question?

Reply:
Hmm, on other projects we have been able to regenerate all of the outputs every time the pipeline has re-run (e.g. for the production files on which thematic analysis was done previously, we just overwrote the existing file with a new one that contained the newest data).

I don't think I know enough about this project to comment, @lukechurch

Reply:
The incremental data was coming from one survey. Due to delivery timelines, we started thematic analysis on Saturday of data from the survey sent on Friday, adding new data on Sunday.